app/assets/stylesheets/submission.css

```diff
@@ -13,6 +13,11 @@ table.results {
   vertical-align: top;
 }
 
+.results .inspect_output th {
+  padding: 0;
+  font-size: 0.95em;
+}
+
```

Comment:
Currently the headings (e.g. "Input") and the stats (e.g. "Size: 2 Bytes") are in separate columns of equal width, which can cause unnecessary word wrapping since each must fit within its own half.
How about we merge the columns and use an alternative way to right-align the stats, for example a \<span\> element with `float:right`? It changes the behaviour when wrapping, but to me it looks fine:
![3-float-right](https://user-images.githubusercontent.com/5887562/75104375-4d81d600-566d-11ea-82b1-964e5e565f17.png)

Reply:
Yeah, I like this idea